tee-worker/client-api/parachain-api/prepare-build/interfaces/identity/definitions.ts

```diff
@@ -41,7 +41,6 @@ export default {
                 "Litmus",
                 "LitentryRococo",
                 "Khala",
-                "SubstrateTestnet",
```

Comment:
You need to fill the gap with __Unused6: "Null",

Reply:
Thank @kziemianek! Updated ✅